waku/node/v2/config.nim

```diff
@@ -139,6 +139,11 @@ type
       desc: "Enable metrics logging."
       defaultValue: false
       name: "log-metrics" }: bool
+      
+    topics* {.
+      desc: "Default topics to subscribe to (space seperated list)."
+      defaultValue: "waku"
+      name: "topics" .}: string
```

Comment:
Optionally you could also use sequence functionality of confutils here. Setting this to `seq[string]` or `seq[Topic]` and changing the name to e.g. singular "topic" would allow to do cli argument like: --topic:waku --topic:bla .